projects/symmetric/index.js

```diff
@@ -19,28 +18,42 @@ async function getTVL(subgraphName, block) {
     }
   }
   `;
-  const results = await graphQLClient.request(query, {
+  var v2 = gql`
+  query get_tvl($block: Int) {
+    balancers(
+      first: 5,
+      block: { number: $block }
+    ) {
+      totalLiquidity,
+      totalSwapVolume
+    }
+  }
+  `;
+  const results = await graphQLClient.request(version === 'v1' ? v1 : v2, {
     block
   })
-  return results.symmetrics[0].totalLiquidity;
+  return version ==='v1'? results.symmetrics[0].totalLiquidity : results.balancers[0].totalLiquidity;
 }
 
-async function xdai(timestamp, ethBlock, chainBlocks) {
-  return toUSDTBalances(await getTVL("symmetric-xdai", await getBlock(timestamp, "xdai", chainBlocks)))
+async function gnosis(timestamp, ethBlock, chainBlocks) {
+  const [v1,v2] = await Promise.all([getTVL("symmetric-xdai", await getBlock(timestamp, "xdai", chainBlocks), 'v1'), getTVL("symmetric-v2-gnosis", await getBlock(timestamp, "xdai", chainBlocks), 'v2')])
+  return toUSDTBalances(BigNumber(v1).plus(v2))
 }
 
 async function celo(timestamp, ethBlock, chainBlocks) {
-  return toUSDTBalances(await getTVL("symmetric-celo", await getBlock(timestamp, "celo", chainBlocks)))
+  const [v1,v2] = await Promise.all([getTVL("symmetric-celo", await getBlock(timestamp, "celo", chainBlocks), 'v1'), getTVL("symmetric-v2-celo", await getBlock(timestamp, "celo", chainBlocks), 'v2')])
+  return toUSDTBalances(BigNumber(v1).plus(v2))
 }
 
 module.exports = {
   misrepresentedTokens: true,
-  methodology: `Symmetric is an Automated Market Maker (AMM) and a Decentralized Exchange (DEX), running on the Celo and xDai networks.
-  Symmetric TVL is pulled from the Symmetric subgraph and includes deposits made to Symmetric xDai and Symmetric Celo liquidity pools.`,
+  methodology: `Symmetric is an Automated Market Maker (AMM) and a Decentralized Exchange (DEX), running on the Celo and Gnosis networks.
+  Symmetric TVL is pulled from the Symmetric subgraph and includes deposits made to Symmetric Gnosis and Symmetric Celo V1 & V2 liquidity pools.`,
   celo:{
     tvl: celo
   },
-  xdai:{
-    tvl: xdai
+  gnosis:{
+    tvl: gnosis
   },
+  tvl: sdk.util.sumChainTvls([gnosis, celo])
```

Comment:
this line can be removed, we no longer need this.

Reply:
Removed.